marlowe-dashboard-client/src/Main.purs

```diff
@@ -64,7 +67,7 @@ main = do
       $ forkAff
       $ WS.runWebSocketManager
           (WS.URI "/ws")
-          (\msg -> void $ driver.query $ ReceiveWebSocketMessage msg unit)
+          (\msg -> void $ forkAff $ driver.query $ ReceiveWebSocketMessage msg unit)
```

Comment:
This one took me a while 😞

Normally when a handler waits on a `AVar` only that handler gets blocked. For normal clicks or subscriptions it's done automatically. But because we are querying Halogen from outside the framework, we also need to fork the aff, to avoid WebService messages getting blocked